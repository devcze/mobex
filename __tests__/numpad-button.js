/* eslint-env jest */

import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import NumpadButton from '../components/numpad-button'

describe('NumpadButton', () => {
  it('NumpadButton render', () => {
    const component = renderer.create(<NumpadButton/>);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
