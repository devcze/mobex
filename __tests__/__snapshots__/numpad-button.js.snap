// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NumpadButton NumpadButton render 1`] = `
<div
  className="numpad-button__NumpadButton-sc-1n447cv-0 fATyam"
  onMouseDown={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  style={
    Object {
      "backgroundColor": "white",
      "color": "#444444",
    }
  }
/>
`;
